modules/nw-ovn-kubernetes-migration-about-live.adoc

```diff
@@ -0,0 +1,166 @@
+= OpenShift SDN live migration overview
+
+The OpenShift SDN live migration is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin. The live migration method offers the following benefits: 
+
+* Continuous service availability
+* Minimized downtime
+* Automatic node rebooting
+* Seamless transition from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin. 
+
+The live migration method is valuable for deployment types that require constant service availability, such as Azure Red Hat OpenShift (ARO), Red Hat OpenShift Dedicated, or Red Hat OpenShift Service on AWS. It is not available for HyperShift deployment types. 
+
+== Considerations for migrating to the OVN-Kubernetes network plugin
+
+* When the live migration procedure begins, multitenant isolation is permanently disabled. Multitenant isolation is only supported on OpenShift SDN. 
+
+* During the live migration, when the cluster is running both OVN-Kubernetes and OpenShift SDN, multicast and egress IP are temporarily disabled for both CNIs.
+
+* Although a rollback procedure is provided, the migration is intended to be a one-way process. For users that want to rollback to OpenShift-SDN, migration from OpenShift-SDN to OVN-Kubernetes must have succeeded. Users can follow the same procedure below to migrate to the OpenShift SDN network plugin from the OVN-Kubernetes network plugin.
+
+* The live migration is only supported on managed clusters. It is not supported on HyperShift clusters. 
+
+== Information about the OpenShift SDN live migration script
+
+The live migration method for migrating from OpenShift SDN network plugin to the OVN-Kubernetes network plugin is a two-step procedure that contains multiple internal processes that are executed by a wrapper script. When running the two commands, the following events occur:
```

Comment:
The live migration is a multi-step procedure that resides in the CNO code.

Reply:
Updated to 

The live migration method for migrating from OpenShift SDN network plugin to the OVN-Kubernetes network plugin is a multi-step procedure that resides in the Cluster Network Operator node. It contains multiple internal processes that are executed by a wrapper script. When running the commands, the following events occur: